js/components/Table/table.scss

```diff
@@ -2,8 +2,7 @@
  * Table
  */
 @import '~@dosomething/forge/scss/toolkit';
-@import './sass/app';
-
+@import '../../../sass/app';
```

Comment:
You shouldn't need to import the `app` SCSS file here. This will include it in your compiled CSS twice - once from [the main entry point](https://github.com/DoSomething/domino/blob/1a4a9592d8aef2f8bc13834381a6e2346914bd0b/js/app.js#L2) and once from here. (This is also why it can't find those font URLs! Webpack resolves `url()` paths from wherever the JavaScript entry point comes from, since everything else is "hidden" inside the SCSS compiler.)

Reply:
ah! 💡 This is so good to know - thank you so much @DFurnes ! I removed that line but now is erroring out saying it can't find `$primary-color` 🤔 

```
Northstar.js:38 Uncaught Error: Module build failed: ModuleBuildError: Module build failed: 
  background-color: $primary-color;
```

Reply:
Oh, right! I'd recommend making a local "toolkit" with variables like that, like we did [here](https://github.com/DoSomething/phoenix-next/blob/master/resources/assets/scss/next-toolbox.scss) in Phoenix Next. (It's important that file only has variables, functions & mix-ins - no outputted CSS!!)

Reply:
oooo I see! I put those in the `app.scss` file but now see it should be separate!